### ISSUE TITLE
Instructions for use

### ISSUE THREAD

**Issue by bkr1969** [Makin-Things/platinum-weather-card#29]
I like the looks of this, but how do we implement it?  
```
custom:platinum-weather-card

```
doesn't work.

**Comment by Makin-Things** [comment]
Have you installed the card using HACS? It is not in Core.

**Comment by DavidFW1960** [comment]
I swear I responded to this 10 minutes ago and now poof...

**Comment by Makin-Things** [comment]
You should not have to use yaml. Once installed thorugh HACS you should be able to just add it using the gui card selector. I am guessing you are in the US and use the NWS integration? If so have a look at this topic starting at the linked post from Pocket. He has done a lot of work to shoehorn the NWS integration with it's quirks. Also please update to 0.0.3 (just released).

https://community.home-assistant.io/t/customising-the-bom-weather-and-lovelace-now-in-hacs/123549/1330?u=therat

**Comment by bkr1969** [comment]
I’m not sure why you think I’m using YAML. In my Lovelace dashboard I’m trying to add a custom card but I don’t know what it’s called as per my original post.
Is it possible to just demonstrate the steps like in all the other HACS cards?

**Comment by DavidFW1960** [comment]
It's down the bottom just above manual cards
![image](https://user-images.githubusercontent.com/36390762/177913043-f55743c2-3b1b-4f8e-a543-2a8f1e1a7744.png)

**Comment by umrath** [comment]
It works fine to install it via HACS.
But adding the card to a view does not yield a usable result. There is no option to choose a weather source. So, it displays error messages only.

**Comment by Makin-Things** [comment]
Just deleted what I typed (because of your edit).
Some weather integrations provide additional sensors that provide better info for some fields, so it is all fully customisable. For now if you just have a standard weather integration that only provides the weather.xxxxx type entity you just need to select that for the fields that it makes sense.
I am thinking about a better way to set that in a more global fashion, but for now that's the way it is.

**Comment by vampywiz17** [comment]
Install it with HACS  (use HASS OS) but not see it on Custom cards... try to restart HA and clrear cache, but not help at all.

**Comment by sesquipedality** [comment]
Yep, seeing this problem too.   I have installed 0.0.3 via HACS and custom:platinum-weather-card does not appear in my list of cards.  It is showing as installed within HACS.  Cannot see anything in the logs that might explain why it's not appearing.

**Comment by bkr1969** [comment]
The concept is good but the implementation is really not even possible.  I stumbled around and decided to try editing the sections.  No matter which section i choose and which sensor I put there, I get this error:
```
'entity_summary_1=sensor.openweathermap_forecast_condition' value needs to have a number
```
I realize this is in beta, but it should really be able to function in some capacity (pass alpha) or there's no way any of us can test it.

Please put code examples somewhere.

**Comment by TerribleTesla** [comment]
Noticed the same thing here.  Installed 0.0.3 from HACS, restarted, cleared cache, etc; card never showed up.

Workaround?  Fix?

Install 0.0.2, and then install 0.0.3 over the top; seemed to work for me
running Version | core-2022.7.1

**Comment by TerribleTesla** [comment]
> Yep, seeing this problem too. I have installed 0.0.3 via HACS and custom:platinum-weather-card does not appear in my list of cards. It is showing as installed within HACS. Cannot see anything in the logs that might explain why it's not appearing.

Noticed the same thing here. Installed 0.0.3 from HACS, restarted, cleared cache, etc; card never showed up.

Workaround? Fix?

Install 0.0.2, and then install 0.0.3 over the top; seemed to work for me
running Version | core-2022.7.1

**Comment by bkr1969** [comment]
I'm current on everything and I don't even see the card as an option now.

**Comment by sesquipedality** [comment]
> Workaround? Fix?
> 
> Install 0.0.2, and then install 0.0.3 over the top; seemed to work for me running Version | core-2022.7.1

Can confirm that installing 0.0.2 then 0.0.3 got the card to appear for me.

**Comment by bkr1969** [comment]
I currently have neither installed and do not find it in the list on HACS to install.

**Comment by DavidFW1960** [comment]
Try downloading it in HACS again and checking the resource is added and correct. Also restart HA. There's some weird things with 2022.7.x I found.

**Comment by bkr1969** [comment]
I cannot download it. It doesn’t show up in the list on HACS.

**Comment by DavidFW1960** [comment]
This could be a hacs error. Simon raised an issue. If you check lovelace resources what do you see for platinum-weather-card?

**Comment by TTLucian** [comment]
Same here. Card does not show up in card selector. Just installed it through HACS.

**Comment by DavidFW1960** [comment]
Same question as above. What is the resource for this card?

**Comment by TTLucian** [comment]
![Screenshot_2022-07-09-08-23-11-03_c3a231c25ed346e59462e84656a70e50.jpg](https://user-images.githubusercontent.com/1855276/178092825-aa2308bc-48cc-45d5-a4b4-17826d219a11.jpg)

**Comment by TTLucian** [comment]
I uninstalled 0.0.3, reinstalled 0.0.2, updated to 0.0.3 and now it shows up

**Comment by DavidFW1960** [comment]
Obviously thats not supposed to happen. There is a hacs issue and we didn't see this happen before 2022.7.0 either

**Comment by bkr1969** [comment]
My issue is completely different. Due to the problems I was having, I uninstalled this card thinking that a re-install would help. Now when I go to HACS and search for “weather” this card is not in the list to install. It just doesn’t exist. I checked my raw files and there isn’t anything leftover from the original install so I’m not sure how I could ever even try to use this card at this point.

**Comment by DavidFW1960** [comment]
A new release of hacs has fixed this

**Comment by Makin-Things** [comment]
Fixed by HACS 1.26. Closing.
